lightningd/plugin.c

```diff
@@ -1011,7 +1012,7 @@ void plugins_init(struct plugins *plugins, const char *dev_plugin_debug)
 	plugins_add_default_dir(plugins,
 				path_join(tmpctx, plugins->ld->config_dir, "plugins"));
 
-	setenv("LIGHTNINGD_PLUGIN", "1", 1);
```

Comment:
Love the approach, but it seems like it'd be better to add a second env called `LIGHTNINGD_VERSION`. As written, `LIGHTNINGD_PLUGIN` is a bool that flags whether or not you're running as a plugin.

Reply:
Fixed it, thanks